gpkit/nomials/nomial_math.py

```diff
@@ -469,6 +451,25 @@ def __init__(self, left, oper, right):
         self.nomials = [self.left, self.right, self.p_lt, self.m_gt]
         self.nomials.extend(self._unsubbed)
 
+    def simplify_posy_ineq(self, exps, cs):
+        "Simplify a posy <= 1 by moving constants to the right side."
```

Comment:
Right now it's just how it's used. My thought was that it would be helpful for the user to know just which constraint was triggering the error.